program/c/src/oracle/upd_aggregate.h

```diff
@@ -134,14 +134,6 @@ static inline void upd_twap(
 // update aggregate price
 static inline bool upd_aggregate( pc_price_t *ptr, uint64_t slot, int64_t timestamp )
 {
-  // Update the value of the previous price, if it had TRADING status.
-  if ( ptr->agg_.status_ == PC_STATUS_TRADING ) {
-    ptr->prev_slot_      = ptr->agg_.pub_slot_;
-    ptr->prev_price_     = ptr->agg_.price_;
-    ptr->prev_conf_      = ptr->agg_.conf_;
-    ptr->prev_timestamp_ = ptr->timestamp_;
-  }
-
   // update aggregate details ready for next slot
   ptr->valid_slot_ = ptr->agg_.pub_slot_;// valid slot-time of agg. price
```

Comment:
feel like we should deprecate valid slot here, this field serves no purpose after this PR

Reply:
ok I see I had already made this comment haha

Reply:
let's leave it for a future PR